common/mapper/src/main/java/io/helidon/common/mapper/MapperManagerImpl.java

```diff
@@ -81,6 +123,10 @@ private RuntimeException createMapperException(Object source,
                     .orElseGet(() -> {
                         GenericType<SOURCE> sourceGenericType = GenericType.create(sourceType);
                         GenericType<TARGET> targetGenericType = GenericType.create(targetType);
+                        if (targetType.isAssignableFrom(sourceType)) {
+                            // the same type
+                            return it -> it;
```

Comment:
Would be nice to have something like `Fuction.identity()` maybe `Mapper.noop()`?

Reply:
Fixed